sdk/cosmosdb/cosmos/src/client/SasToken/Paths.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export class Paths {
+
+  public static ROOT: string = "/";
+
+  public static ROOT_CHAR: string = '/';
+
+  public static ESCAPE_CHAR: string = '\\';
+
+  public static DATABASES_PATH_SEGMENT: string = "dbs";
+
+  public static DATABASES_ROOT: string = (`${Paths.ROOT}${Paths.DATABASES_PATH_SEGMENT}`);
+
+  public static USERS_PATH_SEGMENT: string = "users";
+
+  public static PERMISSIONS_PATH_SEGMENT: string = "permissions";
+
+  public static COLLECTIONS_PATH_SEGMENT: string = "colls";
+
+  public static STORED_PROCEDURES_PATH_SEGMENT: string = "sprocs";
+
+  public static TRIGGERS_PATH_SEGMENT: string = "triggers";
```

Comment:
FYI, You can mostly omit typescript type declarations for something like this.

```suggestion
  public static TRIGGERS_PATH_SEGMENT = "triggers";
```